Azure/sonic-buildimage
# [Celestica] Enable VXLAN support on Seastone2

#### Why I did it
Since the Seastone2 is TD3-based, it supports VXLANs, but the config file did not contain the relevant options yet.
#### How I did it
The configuration is based on https://github.com/Azure/sonic-buildimage/issues/8371#issuecomment-896390372, but some additional options were needed (based on feedback from Celestica) for proper operation.
#### How to verify it
Deploy new config on a Seastone2, follow the steps described in #10050, observe working VXLAN.
#### Which release branch to backport (provide reason below if selected)

#### Description for the changelog
Enable VXLAN support on Seastone2

#### Link to config_db schema for YANG module changes

#### A picture of a cute animal (not mandatory but encouraged)
![image (1)](https://user-images.githubusercontent.com/10601319/172642943-ee15c9c8-ef6d-4a31-8df0-a1c635b97caf.jpg)
